cs3client/file.py

```diff
@@ -183,14 +182,16 @@ def write_file(
         :param resource: Resource to write content to
         :param content: content to write
         :param size: size of content (optional)
-        :param lock_md: tuple (<app_name>, <lock_id>)
+        :param lock_md: tuple (<app_name>, <lock_id>) (optional)
         :return: None (Success)
         :raises: FileLockedException (File is locked),
         :raises: AuthenticationException (Authentication failed)
         :raises: UnknownException (Unknown error)
 
         """
-        app_name, lock_id = lock_md
+        app_name = lock_id = ''
```

Comment:
Is Wopiserver setting lock_md to None here instead of using the default value? because I think that the default value should do the same thing as this if no lock_md is provided. 

I am fine with it either way if having a default value somehow screws things up in the wopiserver.

Reply:
Yep, that's the thing: in wopiserver I always pass the argument, which is either None or the tuple. But you're right this is wopiserver-specific and should be fixed there. I even wonder whether to split this parameter - it's the only API that uses a tuple, all the others don't, and after all the tuple is a trick for wopiserver to deal with the CS3 and the xroot interfaces in an agnostic way.

Reply:
Yeah, I was thinking that I would split this parameter eventually but didn't have time to do it eventually.